src/main/config/messages_template.xml

```diff
@@ -7,413 +7,415 @@ Copyright 2005, 2006 IBM Corporation
+    <response>To remove this message, specify a rule for '%1' in the DITAVAL file.</response>
+  </message>
+
   <message id="DOTJ033E" type="ERROR">
-    <reason>No valid content is found in topicref '%1' during chunk processing.</reason>
-    <response>Please specify an existing and valid topic for the topicref.</response>
-  </message> 
-  
+    <reason>No valid content found in topicref '%1' during chunk processing.</reason>
+    <response>Specify an existing and valid topic for the topic reference.</response>
+  </message>
+
   <!-- 2012-06-12: Still need to reproduce and edit this message -->
   <message id="DOTJ034F" type="FATAL">
-    <reason>Failed to parse the input file '%1' (the content of the file is not valid).</reason>
-    <response>If the input file '%1' does not have a DOCTYPE declaration, please make sure that all class attributes are present in the file.</response>
+    <reason>Failed to parse the input resource '%1' (the content is not valid).</reason>
+    <response>If '%1' does not have a DOCTYPE declaration, make sure that all @class attributes are present.</response>
   </message>
-  
-  <!-- 2012-06-12: Still need to edit this message -->
+
   <message id="DOTJ035F" type="FATAL">
-    <reason>The file "%1" is outside the scope of the input dita/map directory.</reason>
-    <response>If you want to lower the severity level, please use the Ant parameter 'outer.control', and set the value to "warn" or "quiet". Otherwise, move the referenced file "%1" into the input dita/map directory.</response>
+    <reason>The '%1' resource is outside the scope of the input directory.</reason>
+    <response>To lower the severity level, use the Ant parameter 'outer.control', and set the value to 'warn' or 'quiet'. Otherwise, move the '%1' resource to the directory where the input map or topic is stored.</response>
   </message>
-  
+
   <message id="DOTJ036W" type="WARN">
-    <reason>The file "%1" is outside the scope of the input dita/map directory.</reason>
+    <reason>The '%1' resource is outside the scope of the input directory.</reason>
     <response></response>
   </message>
-  
-  <!-- 2012-06-12: Still need to edit this message -->
+
   <message id="DOTJ037W" type="WARN">
     <reason>The XML schema and DTD validation function of the parser is turned off.</reason>
-    <response>Please make sure the input is normalized DITA with class attributes included, otherwise it will not be processed correctly.</response>
+    <response>For correct processing, make sure the input is normalized DITA with @class attributes included.</response>
   </message>
-  
+
   <message id="DOTJ038E" type="ERROR">
-    <reason>The tag "%1" is specialized from unrecognized metadata.</reason>
-    <response>Please make sure that tag "%1" is specialized from an existing metadata tag in the core DITA vocabulary.</response>
+    <reason>The &lt;%1&gt; element is specialized from unrecognized metadata.</reason>
```

Comment:
Is this one we can mark deprecated? My search of our code for `038` only turns up a different XSLT message, and this definition. At this point I'm not sure what it would mean to specialize from "unrecognized metadata" so I'm not sure what this was doing.

Reply:
That sounds like a question for @jelovirt.

Reply:
(Marked as deprecated pending further info.)

Reply:
@infotexture I can't find use for this anywhere. Looks like an unused message now.

Reply:
@jelovirt OK, thanks for confirming. 

I marked it as deprecated in 89a6446, so this should be ready for review & merge.